packages/core/src/components/dialog/dialog.md

```diff
@@ -39,23 +39,12 @@ class DialogExample extends React.Component<{}, IDialogExampleState> {
         return (
             <div>
                 <Button onClick={this.toggleDialog} text="Show dialog" />
-                <Dialog
-                    iconName="inbox"
-                    isOpen={this.state.isOpen}
-                    onClose={this.toggleDialog}
-                    title="Dialog header"
-                >
-                    <div className="pt-dialog-body">
-                        Some content
-                    </div>
+                <Dialog icon="inbox" isOpen={this.state.isOpen} onClose={this.toggleDialog} title="Dialog header">
+                    <div className="pt-dialog-body">Some content</div>
                     <div className="pt-dialog-footer">
                         <div className="pt-dialog-footer-actions">
                             <Button text="Secondary" />
-                            <Button
-                                intent={Intent.PRIMARY}
-                                onClick={this.toggleDialog}
-                                text="Primary"
-                            />
+                            <Button intent={Intent.PRIMARY} onClick={this.toggleDialog} text="Primary" />
```

Comment:
these lines (42 and 47) are too wide for the code sample box; revert to multiline syntax.

![image](https://user-images.githubusercontent.com/464822/35949207-2c95f172-0c25-11e8-904c-c62dbcb1ae8a.png)